### PR TITLE
28: Roles Definition

### PR DESCRIPTION
**Pull request by robherba**
**Steps to test**
- Check the new roles in file: profiles/manatime/manatime.install.

[Issue link](https://github.com/ManatiCR/manatime/issues/28)

### CODE REVIEW

**Comment by robherba** [comment]
Please check it.

File: profiles/manatime/manatime.install [hunk 1]
```diff
@@ -11,7 +11,9 @@ function manatime_install() {
   // Create a default role for site administrators,
   // with all available permissions assigned.
   $admin_role = new stdClass();
+  $admin_role->name = 'super admin';
   $admin_role->name = 'administrator';
+  $admin_role->name = 'collaborator';
```
Comment (kporras07) [inline profiles/manatime/manatime.install]: You need a new stdClass using a variable with a meaningful name (you can also use a loop) and a user_role_save call per each role.

**Comment by kporras07** [comment]
btw, you don't set Needs Review label unless you think the PR is ready to be reviewed :)

**Comment by robherba** [comment]
Changes has been applied.

**Comment by robherba** [comment]
@kporras07

File: profiles/manatime/manatime.install [hunk 2]
```diff
@@ -10,17 +10,30 @@
 function manatime_install() {
   // Create a default role for site administrators,
   // with all available permissions assigned.
+  $super_admin_role = new stdClass();
```
Comment (kporras07) [inline profiles/manatime/manatime.install]: It's working now; however, by readability, I'd prefer this order:
- $super_admin_role = new stdClass();
- $super_admin_role->name = 'super admin';
- $super_admin_role->weight = 2;
- user_role_save($super_admin_role);

Then the same for every other role.

File: profiles/manatime/manatime.install [hunk 3]
```diff
@@ -10,17 +10,30 @@
...
   $admin_role = new stdClass();
+  $collaborator_role = new stdClass();
+
+  $super_admin_role->name = 'super admin';
   $admin_role->name = 'administrator';
+  $collaborator_role->name = 'collaborator';
+
+  $super_admin_role->weight = 2;
   $admin_role->weight = 2;
+  $collaborator_role->weight = 2;
```
Comment (kporras07) [inline profiles/manatime/manatime.install:16]: Weight 2 for collaborator is ok.

File: profiles/manatime/manatime.install [hunk 4]
```diff
@@ -10,17 +10,30 @@
...
+  $super_admin_role = new stdClass();
   $admin_role = new stdClass();
+  $collaborator_role = new stdClass();
+
+  $super_admin_role->name = 'super admin';
   $admin_role->name = 'administrator';
+  $collaborator_role->name = 'collaborator';
+
+  $super_admin_role->weight = 2;
   $admin_role->weight = 2;
```
Comment (kporras07) [inline profiles/manatime/manatime.install]: Use weight 3 for admin.

File: profiles/manatime/manatime.install [hunk 5]
```diff
@@ -10,17 +10,30 @@
...
   // with all available permissions assigned.
+  $super_admin_role = new stdClass();
   $admin_role = new stdClass();
+  $collaborator_role = new stdClass();
+
+  $super_admin_role->name = 'super admin';
   $admin_role->name = 'administrator';
+  $collaborator_role->name = 'collaborator';
+
+  $super_admin_role->weight = 2;
```
Comment (kporras07) [inline profiles/manatime/manatime.install]: Use weight 4 for super admin.

**Comment by kporras07** [comment]
@rohernandez it works; just a couple minor changes.

File: profiles/manatime/manatime.install (hunk 2 above)
Comment (robherba) [reply profiles/manatime/manatime.install]: Changed!

File: profiles/manatime/manatime.install (hunk 3 above)
Comment (robherba) [reply profiles/manatime/manatime.install:16]: ok

File: profiles/manatime/manatime.install (hunk 4 above)
Comment (robherba) [reply profiles/manatime/manatime.install]: Changed!

File: profiles/manatime/manatime.install (hunk 5 above)
Comment (robherba) [reply profiles/manatime/manatime.install]: Changed!!

**Comment by robherba** [comment]
Everything is ready!

**Comment by robherba** [comment]
@kporras07

**Comment by kporras07** [comment]
@rohernandez :shipit:

**Pull request merged**